pages/features/auto-responder.vue

```diff
@@ -204,25 +248,10 @@ export default {
               position: 2,
               name: 'Auto Responder',
               item: 'https://formester.com/features/auto-responder/',
-            }
+            },
           ],
         },
       ],
-      address: {
-        '@type': 'PostalAddress',
-        addressLocality: 'Dalaware',
-        addressCountry: 'United States',
-      },
-      creator: {
-        '@type': 'Organization',
-        '@id': '#organization',
-        url: 'https://formester.com/',
-        name: 'Formester',
-        logo: {
-          '@type': 'ImageObject',
-          url: 'https://formester.com/logo.png',
-        },
-      }
```

Comment:
Are these not required anymore?

Reply:
There were duplicated entries for address and creator in some places, hence removed them. Also, 'creator' is not required in organisation schema, hence removed that too.

Reply:
okay